test/client-deselect.js

```diff
@@ -0,0 +1,109 @@
+import fixtures from 'webtorrent-fixtures'
+import MemoryChunkStore from 'memory-chunk-store'
+import test from 'tape'
+import WebTorrent from '../index.js'
+
+function setupClient ({ t, onTorrent, onDone, addTorrentProps = {} }) {
+  const client1 = new WebTorrent({ dht: false, tracker: false, lsd: false })
+  client1.on('error', function (err) { t.fail(err) })
+  client1.on('warning', function (err) { t.fail(err) })
+
+  const client2 = new WebTorrent({ dht: false, tracker: false, lsd: false })
+  client2.on('error', function (err) { t.fail(err) })
+  client2.on('warning', function (err) { t.fail(err) })
+
+  const parsedTorrent = Object.assign({}, fixtures.leaves.parsedTorrent)
+
+  client1.seed(fixtures.leaves.content, {
+    name: 'Leaves of Grass by Walt Whitman.epub',
+    announce: [],
+    store: MemoryChunkStore
+  }, () => {
+    client2.add(parsedTorrent, { store: MemoryChunkStore, ...addTorrentProps }, (torrent) => {
+      onTorrent(torrent)
+
+      torrent.addPeer(`localhost:${client1.torrentPort}`)
```

Comment:
you're adding a peer by IP, this is a no-go in browser, move this test to test/node